app/ui/shared/widgets/autocompleted-search.jsx

```diff
@@ -47,20 +47,26 @@ class AutocompletedSearch extends Component {
 
   componentWillMount() {
     if (typeof window === 'object') {
-      window.addEventListener('click', this.handleWindowClick);
+      window.addEventListener('mousedown', this.handleWindowMousedown);
     }
   }
 
   componentWillUnmount() {
     if (typeof window === 'object') {
-      window.removeEventListener('click', this.handleWindowClick);
+      window.removeEventListener('mousedown', this.handleWindowMousedown);
     }
   }
 
-  handleWindowClick = () => {
-    // We can't call `setState` on components which haven't rendered yet.
-    if (this.state.showSelectionList && this.inputbar) {
+  // We need to use mousedown since preventing the event on click won't stop
+  // it from passing through to the webview.
```

Comment:
This is just from my testing, I haven't found it documented anywhere.  But if you preventDefault() on a click event that happened in a webview it doesn't stop it from happening on the page but if you preventDefault() on a mousedown it does.

stopPropagation shouldn't help here - the event target will be something in the webview and it's already bubbled all the way up to the window by now.